### PR TITLE
Add note concerning Podman/MacOS combination; 

### LINKED ISSUE #431: Using Podman on MacOS is not supported
Opening this issue for the sake of updating the docs accordingly.

### PR DESCRIPTION
**Pull request by wirednkod**
Fixes #431

### CODE REVIEW

**Review by pepoviola — APPROVED** [review]
Thanks @wirednkod !!

**Comment by wirednkod** [comment]
I would love @sacha-l to have a look on this and apply his awesome skills before merging! 🙏🏽

File: docs/src/requirements/podman.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
...
 
-Zombienet supports [Podman](https://podman.io/) *rootless* as provider.
-You only need to have `podman` installed in your environment to use it and either set it in the *network* file or with the `--provider` flag in the CLI.
+Zombienet supports [Podman](https://podman.io/) _rootless_ as provider.
+You only need to have `podman` installed in your environment to use it and either set it in the _network_ file or with the `--provider` flag in the CLI.
 
+**Note:** `Podman` is supported in MacOS as an app but this is happening through an
+internal vm. The provider's code in zombienet supports `podman` running native - thus `podman` for
+`zombienet` can be used only on linux. This is related, mostly, to paths and directories used by
+store configuration (chain-spec) and the data directory.
```
Comment (sacha-l) [inline docs/src/requirements/podman.md:9]:
```suggestion
**Note:** Currently, it is only possible to use `podman` for Zombienet users on Linux machines. 
Although `podman` comes with support for macOS, it is done using an internal VM and the Zombienet provider code expects `podman` to be running natively.
```

File: README.md [hunk 2]
```diff
@@ -34,6 +39,11 @@ Zombienet releases are available in `github`. Each one provides an executable fo
 *without* having `Node.js` installed **but** each `provider` defines it's own requirements (e.g.
 `k8s`, `podman`).
 
+**Note:** `Podman` is supported in MacOS as an app but this is happening through an
+internal vm. The provider's code in zombienet supports `podman` running native - thus `podman` for
+`zombienet` can be used only on linux. This is related, mostly, to paths and directories used by
+store configuration (chain-spec) and the data directory.
```
Comment (sacha-l) [inline README.md:45]:
```suggestion
**Note:** Currently, it is only possible to use `podman` for Zombienet users on Linux machines. 
Although `podman` comes with support for macOS, it is done using an internal VM and the Zombienet provider code expects `podman` to be running natively.
```

File: README.md [hunk 3]
```diff
@@ -25,7 +25,12 @@ write tests as smooth as posible.
 
 Internally zombienet is a `javascript` library, designed to run on `Node.js` and support different
 backend `providers` to run the *nodes*, at this moment `kubernetes`, `podman` and `native` are
-supported.
+supported. 
+
+**Note:** `Podman` is supported in MacOS as an app but this is happening through an
+internal vm. The provider's code in zombienet supports `podman` running native - thus `podman` for
+`zombienet` can be used only on linux. This is related, mostly, to paths and directories used by
+store configuration (chain-spec) and the data directory.
```
Comment (sacha-l) [inline README.md:33]:
```suggestion
**Note:** Currently, it is only possible to use `podman` for Zombienet users on Linux machines. 
Although `podman` comes with support for macOS, it is done using an internal VM and the Zombienet provider code expects `podman` to be running natively.
```

File: README.md [hunk 4]
```diff
@@ -58,7 +68,9 @@ Zombienet project has it's own `k8s` cluster in GCP, to use it please ping
 
 Zombienet support [Podman](https://podman.io/) *rootless* as provider, you only need to have
 `podman` installed in your environment to use and either set in the *network* file or with the
-`--provider` flag in the cli.
+`--provider` flag in the cli. `Podman` for `zombienet` can be used only on linux even though `podman`
+is an app supported also on MacOS. This is related, mostly, to paths and directories used by 
+store configuration (chain-spec) and the data directory.
```
Comment (sacha-l) [inline README.md:73]:
```suggestion
`--provider` flag in the cli. `Podman` for `zombienet` is currently only supported for Linux machines.
This is mostly related to paths and directories used by 
store configuration (chain-spec) and the data directory.
```

**Review by sacha-l — COMMENTED** [review]
Wording and meaning more concise :)

**Pull request merged**